docs/content/Cube.js-Introduction.md

```diff
@@ -1,41 +1,75 @@
+further be tuned for performance.
 
-Unlike others, it is not a monolith application, but a set of modules, which does one thing well. Cube.js provides modules to run transformations and modeling in data warehouse, querying and caching, managing API gateway and building UI on top of that.
+Cube.js provides modules to run transformations and modeling in data warehouses,
+along with querying and caching, managing API gateway and building UI.
 
 ### Cube.js Backend
 
-- __Cube.js Schema.__ It acts as an ORM for analytics and allows to model everything from simple counts to cohort retention and funnel analysis.
-- __Cube.js Query Orchestration and Cache.__ It optimizes query execution by breaking queries into small, fast, reusable and materialized pieces.
-- __Cube.js API Gateway.__ It provides idempotent long polling API which guarantees analytic query results delivery without request time frame limitations and tolerant to connectivity issues.
+- **Schema.** Acts as an ORM for analytics and allows modelling everything from
+  simple counts to cohort retention and funnel analysis
+- **Query Orchestration and Cache.** Optimizes query execution by breaking
+  queries into small, fast, reusable and materialized pieces
+- **API Gateway.** Provides an idempotent long polling API as well as a
+  WebSockets API which guarantees delivery of analytical query results without
+  request timeframe limitations and tolerance to connectivity issues
 
 ### Cube.js Frontend
 
-- __Cube.js Javascript Client.__ Core set of methods to access Cube.js API Gateway and to work with query result sets.
-- __Cube.js React, Angular and Vue.__ Framework specific wrappers for Cube.js API.
+- **Javascript Client.** Core SDK for accessing the Cube.js API Gateway and
+  functionality for working with query result sets
+- **React, Angular and Vue.** Framework specific wrappers for Cube.js API
 
 ## Why Cube.js?
 
-If you are building your own business intelligence tool or customer-facing analytics most probably you'll face following problems:
+If you are building your own business intelligence tool or customer-facing
+analytics, it is quite likely you'll face one or more of the following problems:
 
-1. __Performance.__ Most of effort time in modern analytics software development is spent to provide adequate time to insight. In the world where every company data is a big data writing just SQL query to get insight isn't enough anymore.
-2. __SQL code organization.__ Modelling even a dozen of metrics with a dozen of dimensions using pure SQL queries sooner or later becomes a maintenance nightmare which ends up in building modelling framework.
-3. __Infrastructure.__ Key components every production-ready analytics solution requires: analytic SQL generation, query results caching and execution orchestration, data pre-aggregation, security, API for query results fetch, and visualization.
+1. **Performance.** A significant amount of time in modern analytics software
```

Comment:
This first sentence is a bit awkward (esp. with the word "time" used twice). How about "A significant effort in modern analytics software development is on providing adequate time to insight"?